src/fping.c

```diff
@@ -123,7 +123,7 @@ extern int h_errno;
 #endif
 
 #if !defined(CLOCKID)
-#if defined(CLOCK_MONOTONIC)
+#if defined(CLOCK_MONOTONIC) && !defined(__APPLE__) && !defined(__OpenBSD__) && !defined(__FreeBSD__)
```

Comment:
Could you add a comment to explain why we can't use CLOCK_MONOTONIC on these platforms?

Reply:
On FreeBSD and OpenBSD CLOCK_MONOTONIC begin with 0 after reboot or undefined positive point.
Here is my test result with FreeBSD.
```
CLOCK_MONOTONIC (sec): 43
CLOCK_REALTIME (sec): 1628355889
CLOCK_UPTIME (sec): 43
```
OpenBSD Manpage: https://man.openbsd.org/clock_gettime.2

macOS unfortunately I can not test only assume

Reply:
Thanks, I meant also a comment in the code. Maybe referencing just this PR, something like this:

// don't use CLOCK_MONOTONIC on Mac and FreeBSD because gettime returns uptime (see github PR #217)

Reply:
Ups, sorry for misunderstanding. I have add a in code comment

Reply:
Thanks!